src/Symfony/Component/Notifier/Bridge/Bluesky/composer.json

```
@@ -26,6 +26,9 @@
         "symfony/notifier": "^7.1",
```

```suggestion
        "symfony/notifier": "^7.2",
```
It will not be part of Symfony 7.1